bib/cmd/bootc-image-builder/main.go

```diff
@@ -344,7 +344,7 @@ func run() error {
 	logrus.SetLevel(logrus.ErrorLevel)
 	buildCmd.Flags().AddFlagSet(manifestCmd.Flags())
 	buildCmd.Flags().String("output", ".", "artifact output directory")
```

Comment:
Just to note here, this continues to work because of the new `WORKDIR /output` in the container.